### ISSUE TITLE
Validator function ?

### ISSUE THREAD

**Issue by maxerbox** [z-brain/typed-node-env#1]
It would be great if we could have custom validator functions

**Comment by korniychuk** [comment]
Good idea! However, I don't have time and it's not required for my project.
Maybe one day I'll add.
